risc0/zkvm/src/guest/env/batcher.rs

```diff
@@ -59,8 +59,9 @@ impl Keccak2Batcher {
             let po2: u32 = po2.parse::<u32>().unwrap();
             if !KECCAK_PO2_RANGE.contains(&(po2 as usize)) {
                 panic!(
-                    "invalid keccak po2 {po2}. Expected range: {:?}",
-                    KECCAK_PO2_RANGE
+                    "invalid keccak po2 {po2}. Expected range: {} - {}",
+                    KECCAK_PO2_RANGE.start,
+                    KECCAK_PO2_RANGE.end - 1
```

Comment:
```suggestion
                    "invalid keccak po2 {po2}. Expected range: {:?}",
                    KECCAK_PO2_RANGE
```

If we use `RangeInclusive` does that make this error message sane?

Reply:
I got this:
```
called `Result::unwrap()` on an `Err` value: Guest panicked: invalid keccak po2 19. Expected range: 14..=18
```
I think it looks cleaner if I have the range (but drop the -1 thing). I changed it to look like this now:
```
called `Result::unwrap()` on an `Err` value: Guest panicked: invalid keccak po2 19. Expected range: 14 - 18 (inclusive)
```